### PR TITLE
[subscriberdb][magmad] add print GRPC to other RPC servicers

### PR DESCRIPTION
**Pull request by uri200**
Signed-off-by: Oriol Batalla <obatalla@fb.com>

## Summary

Added some more GRPC printing messages from magmad `rpc_sync_grpc` and from `s6a_proxy_servicer.py` on subscriberdb

## Test Plan

fab integ_test

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/python/magma/subscriberdb/rpc_servicer.py [hunk 1]
```diff
@@ -110,22 +101,22 @@ def ListSubscribers(self, request, context):  # pylint:disable=unused-argument
...
-        try:
-            self._print_grpc(request)
-        except Exception as e:  # pylint: disable=broad-except
-            logging.debug("Exception while trying to log GRPC: %s", e)
+        self._print_grpc(request)
         sids = self._store.list_subscribers()
         sid_msgs = [SIDUtils.to_pb(sid) for sid in sids]
         return subscriberdb_pb2.SubscriberIDSet(sids=sid_msgs)
 
     def _print_grpc(self, message):
```
Comment (themarwhal) [inline lte/gateway/python/magma/subscriberdb/rpc_servicer.py:109]: Can we pull these into a single file so we don't have to keep defining it? Other than that LGTM!

**Review by themarwhal — APPROVED** [review]

File: lte/gateway/python/magma/subscriberdb/rpc_servicer.py (hunk 1 above)
Comment (uri200) [reply lte/gateway/python/magma/subscriberdb/rpc_servicer.py:109]: yep, that is in my backlog. I just don't have the time right now  😭 , but will be done.

**Review by emakeev — APPROVED** [review]

**Review by ardzoht — APPROVED** [review]

**Review by hcgatewood — APPROVED** [review]
+1 to @themarwhal's comment, otherwise lgtm

**Comment by uri200** [comment]
Hey @themarwhal  it seems this PR is stuck on `mme_test` and `session_manager_test`. I rebased yesterday but it seems the results are never available

**Comment by themarwhal** [comment]
Hey @uri200 I think you might have to rebase against the latest master. I took a look at your branch (https://github.com/uri200/magma/blob/subscriberdb_magmad_more_grpc_printing_messages/.circleci/config.yml) and it doesn't contain the latest circleci changes that added those test jobs.

**Pull request merged**